test/test.mk

```diff
@@ -1,6 +1,8 @@
-base-dir := $(lastword $(subst :, ,$(BR2_EXTERNAL)))
-test-dir := $(BR2_EXTERNAL_INFIX_PATH)/test
-ninepm   := $(BR2_EXTERNAL_INFIX_PATH)/test/9pm/9pm.py
+base-dir           := $(lastword $(subst :, ,$(BR2_EXTERNAL)))
+test-dir           := $(BR2_EXTERNAL_INFIX_PATH)/test
+ninepm             := $(BR2_EXTERNAL_INFIX_PATH)/test/9pm/9pm.py
+spec-dir           := $(test-dir)/spec
+test-specification := $(spec-dir)/test-specification.pdf
```

Comment:
This will not be checked in, right? So should it live somewhere in output? `images/`?